src/app/clients/form/submitFormHelpers.ts

```diff
@@ -46,11 +50,14 @@ export const formatClientRecord = (
         address_county: fields.addressCounty,
         address_postcode: fields.addressPostcode,
         default_list: fields.listType as ListType,
-        dietary_requirements: checkboxGroupToArray(fields.dietaryRequirements),
-        feminine_products: checkboxGroupToArray(fields.feminineProducts),
+        dietary_requirements:
+            fields.dietaryRequirements !== null
+                ? checkboxGroupToArray(fields.dietaryRequirements, dietaryRequirementOptions)
```

Comment:
I don't think that this needs to sort the fields because it's just what's going into the database - better for checkboxGroupToArray() to just convert & not sort. Leave the canonical sorting to be only for UI